docs/sdk-scheme-adapter-outbound-v2_0_0-openapi3-snippets.yaml

```diff
@@ -383,15 +383,15 @@ paths:
           $ref: '#/components/responses/transferServerError'
         '504':
           $ref: '#/components/responses/transferTimeout'
-  /requestToPayTransfer/{requestToPayTransactionId}:
+  /requestToPayTransfer/{transactionRequestId}:
     put:
       summary: >-
         Continues a transfer that has paused at the otp stage in order to accept
```

Comment:
```suggestion
        Continues a transfer that has paused at the Authentication stage in order to accept
```

Reply:
It would be really great if we could make this more generic, and less "OTP" specific.